sections/method/Clustering-PageRank.tex

```diff
@@ -0,0 +1,56 @@
+\section{Clustering Random Walk}
+To perform Random Walk on a document set, such as Nordjyske, we need a similarity measure between each document.
+This is calculated by using the document-topic distributions from the \gls{LDA}.
+These distributions can be arranged in a matrix $\theta$ which is described later.
+
+In \cite{ClusterPageRank}, they describe the \gls{Cluster-CMRW} which is new version of the Random Walk model. 
+They improve upon this model by incorporating information from clusters. 
+The clusters are found by employing three different clustering algorithms.
+They create a new transition probability matrix where the cluster information is added by combining three similarity functions based on the information levels present in the data.
+They describe three similarity levels between:
+\begin{itemize}
+    \item Sentence to Sentence - $f(v_i \rightarrow v_j)$
+    \item Sentence to Topic Cluster - $\omega(v_i, clus(v_i))$
+    \item Topic Cluster to Document Set - $\pi(clus(v_i))$
+\end{itemize}
+where $v_i$ is the given sentence and $clus(v_i)$ is the cluster that $v_i$ belongs to.
+
+We want to use this method to incorporate the information given by the topic distributions within a \gls{Cluster-CMRW}. 
+We first create three similarity functions to incorporate topic cluster level information into a new adjacency matrix.
+\begin{itemize}
+    \item Document to Document - $d2d(d_1, d_2)$
+    \item Document to Topic Cluster - $d2t(d,t)$
+    \item Topic Cluster to Document Set - $t2D(t)$
```

Comment:
Vi bruger ikke samme format som i den tidligere itemize. Er der en grund til det?

Reply:
Det er for at sige at det er anderledes end hvad det er i den originale artikel

Reply:
Ok, men bliver man ikke forvirret når man kommer ned til den store similarity funktion (linje 36+), når den bruger samme format som artiklen, med f.eks. `clus(d_1)`, som her er `t`?